nntrainer/layers/common_properties.cpp

```diff
@@ -333,6 +333,17 @@ void GenericShape::set(const TensorDim &value) {
   Property<TensorDim>::set(ret);
 }
 
+NumHeads::NumHeads(unsigned int value) { set(value); }
+
+ProvideAttentionMask::ProvideAttentionMask(bool value) { set(value); }
+
+ReturnAttentionWeight::ReturnAttentionWeight(bool value) { set(value); }
+
+ReturnAttentionWeightBeforeDropout::ReturnAttentionWeightBeforeDropout(
+  bool value) {
```

Comment:
What if those two properties set at the same time?

Reply:
Discussed offline to use only ReturnAttentionWeight property.
ReturnAttentionWeight will be string type property which will have one of "empty", "before", "after" value.